readPass/index.js

```diff
@@ -77,19 +76,42 @@ exports.handler = async (event, context) => {
       }
       const token = jwt.sign(claims, process.env.JWT_SECRET, { expiresIn: '15m' });
 
-      // Redacted passData
-      const redacted = {
-        registrationNumber: passData.data[0].registrationNumber,
-        token: token
-      };
-      
-      // TODO: ********SEND EMAIL INSTEAD
-      // Public page after 200OK should show 'check your email'
-    
+      const cancellationLink = process.env.PUBLIC_FRONTEND
+      + process.env.PASS_CANCELLATION_ROUTE
+      + "?passId=" + passData.data[0].registrationNumber
+      + "&email=" + passData.data[0].email
+      + "&code=" + token;
 
```

Comment:
Indentation 2sp here.  Also - need to check .data array for being empty array incase the query fails for some reason. We don't want to crash node here.